src/Jackalope/Transport/DoctrineDBAL/CachedClient.php

```diff
@@ -259,7 +263,7 @@ public function getNode($path)
         $cacheKey = "nodes: $path, ".$this->workspaceName;
         $cacheKey = $this->sanitizeKey($cacheKey);
 
-        if (false !== ($result = $this->get('nodes', $cacheKey))) {
+        if (null !== ($result = $this->get('nodes', $cacheKey))) {
```

Comment:
why does this change from false to null?

Reply:
because PSR16 return `null` not false when [there is no result](https://github.com/php-fig/simple-cache/blob/2d280c2aaa23a120f35d55cfde8581954a8e77fa/src/CacheInterface.php#L18).

The current code was for the doctrine/cache (returning false), instead of converting null to false I go with converting false to null in the `get` method as @flogado mentiont that `2.x` does not have the `::get` method and always us the PSR16 implementation which returns `null` and not false.

Reply:
ah, now i get it. thanks, yeah better do it this way around.